engine/dlib/src/dlib/pprint.cpp

```diff
@@ -48,11 +48,7 @@ namespace dmPPrint
         }
 
         int c = m_BufferSize - m_Cursor;
-    #if defined(_WIN32)
-        _vsnprintf_s(m_Buffer + m_Cursor, c, _TRUNCATE, format, argp);
-    #else
```

Comment:
Uncommented change?

Reply:
I've updated the PR description. I originally wrote the reasoning behind the change and linked to references in the task, as I was thinking the reviewer would first read it and then the PR. But I see how it's better to have the info here in the PR.

I've rebased to latest dev and added a change: replaced an instance of `_vsnprintf_s` in script_table.cpp that I missed.

Reply:
I see no links in your description though.
Where is the support of the `vsnprintf` mentioned?